packages/overlay/src/index.tsx

```diff
@@ -107,19 +107,22 @@ export async function init(options: SpotlightOverlayOptions = {}) {
 
   const {
     openOnInit = false,
-    showTriggerButton = true,
-    injectImmediately = false,
     sidecarUrl = DEFAULT_SIDECAR_URL,
     anchor = DEFAULT_ANCHOR,
-    debug = false,
     integrations,
     experiments = DEFAULT_EXPERIMENTS,
-    fullPage = false,
     showClearEventsButton = true,
     initialEvents = undefined,
     startFrom = undefined,
   } = options;
 
+  const isLoadedFromSidecar = new URL(sidecarUrl).origin === document.location.origin;
```

Comment:
[NOT RELATED TO THIS PR]

Looking at these, I think we can give the support of query params to users for changing config of spotlight when running overlay via sidecar url. 
Something like 'localhost:8969?integrations=sentry,console&fullScreen=true'

Reply:
Aha, pretty good idea! I wonder if we can do something similar with the Electron app too. For that we may have to fall back to config files :(